Packs/CortexAttackSurfaceManagement/Scripts/GenerateASMReport/GenerateASMReport.py

```diff
@@ -48,402 +50,657 @@ def build_template(args: Dict[str, Any]) -> List[Dict[str, Any]]:
FHOcYo/RxdcFxZhNgbq8DqK28AwfwcuXQmPIP/XffYA/iE+12oVY6bT2gEzHNcMRRAEQRAEQRAEQRAEQf66KU7rhpK9mepUP90Rw2Otxi1I9Zg+N9l9xZxUd/78VE/XxWneF5ak7Q9bnn4wC7Rvl6b6Vi5K9WlenX64ZfX9w80bHhxr2Zpxqnnn43MtvCeXavWyrxYb59yINc++ft0k+5qPec4NQ8PsK+us827Nty+4M3n94yNjeLmXRmjd2ac+dc/absfzvfnRdcME/KFpACnDvO2fAM+waB2T5Lh/xt71GtLn8rXN2btR/V75E6PowketUc8f5nlEnWN2rvvRed0wp7Sr6v5vHmiIX2WMi6p8+mNkxZMF8dUFq2Pe5vDiK/L2PqwrPv2gtigko64kHSjKqCstz6opq31YW9L6tKas9VF9WeuTmrKPj2rL6rNqSqvBNtkPa4sDEysLfCJfPbGIKM/Svvv2ySKw758C3z4cdbosYfgPruuHSR/n/8p4t43DKOB3C3fTHUHaLRhFsjMYg9sajMftjCaR7Ax/ItkbTCM5AHaGU0l2Rj+S7QwnkGwNxpEc9EdjznoahBd3uJrnNvWf9q7rtu+uNh81nhpblnUu5tmjtqj8+w13yzJO6V1xmia93f/Sj87rhyl5aan3EPJH0UO4k6hhvFnkYPZqahifTw7muALXiWB2LPAIeIIHs/OBQnA/Gz4G7gP3gFCwrT8Rwj6Ph7AOgsf7iFCOFRHM0aeIOUwimKVJjeCvpIfwFsDXoIVwpzJCeZMU7wrGKUXojWac0xvV56a5huI5oxFkb5Y64ckfTvFhqk9w2/Dd/Nz8p6btXjt0IjifgR5bhpG9OOraseeHT7vhMXxwmOWIXpEGo+D7oBipPx68N5OBH4HptFDeHFoYbxE1lLuyV7jR+h6hgm3g/WOp3jUz6hNlZqecYOHWJ8z4qFq4yUm1SNPr8mF6ooFRFpH9Is0S+0SaPATLCkBb1OuucUXvUOMq0FbBIdz7Rpp9GBhlCduP/aMsPkBDo61bhsa0Gxa964NGrG0NvA/a6pGxtpVj4uxLgcIRsTZpI+/Z3Z2Y4HQNOD0pcbfHtCRn6wlxTpwhkVaaPyXuWznxnsNsRhh/Gvx8+8eajOp93XQE2YutPtRj8zD4Pki/N8jnYH9J99mpPirFVkM10nDMknSfn6Yl7J07J8V98/L7vrvA9f/k4jQf0exkt6SZKW7Zi9J8Xi9I9X4PNENL0nxbVqQfbF4K2mVpvs2LU/fXwb8TlqX7Fiy9fzAObH9jaZrPwXUPjjgsT/flzk3xXM7JOj/7l3S3CX3vGI3o77l12E97uvdv0a+zHKMLMz7EFmcWnb0vWi+9/lv48QxnzqBkW3HvB/atKum2rX0eOrSqZTj+oT4ZThLw/qDMva39Hjm1Ds5yaYOPh2Y5tw554to2OHMfWL4b3Hdpg8u6guu7tjBUl6wDj2EgPyzLpeO+m+S+OlgGt52Vd1iy7bBMl7pJwVYu0ufxtYx23zQM9pEYvH45CUaTbA0nkuzBtcreYCbmYDgLXLemfbpu2RuMJu3TGynnwR4+2bn7NbinK1MdrNegBrb3/biYvZUQs+2xENY50N4j32I+o17c8Ra/pv0eu6rdSrqyvZV0aXsrdmtHK+ni9vbHpzRbScc3tuL7134g+2yopR7aUso4pp0ue4ZzlXKLvZci4rAZIbxfSQ5GPyqGgr7egzt8kkv3Y0GQ/6tm2q7u+Vl4DucMGuG3bQZ2Xy+QlMZ5iCAIgiAIgiAIgiAIgvx9WBrnCZ7KLYCIVG45OZVbAdHS9VqADzJJJm30NMFHRrJxq0y64Uf5FPM2qEeaZRv1gVkz9tC0EXhDTuVVUNL4+R37ypN+nX8KLI2bRU3X/Qjut/0TUMCxgmN+Qvqdc/nawOtkDHpsXz8w0/FD7wzLcun10uDPAvmhaT4kl2FRDJ5b2xXYT8v41D0fhzxyavshec/HQZlObRNBOxQ8Hpu6u7lzO8UMqxed+8HS+Y+lX+ffBEvnPZLJsCgdCN6rofcdWnsnmVbC32Hp7b5TGR39Q1FnP/MXvQJ9UzHwsrOv6gT2+Qi0+b/zWshfAH92gCwslZsj/d4Cz8B7/5r4nc+LmqbbSEnl1dDSdZtl7+pKrhWSNlXwkRwvaMOT+W1Egl4bkajbRksSfITrJdeTZJM2WrpAcv/3gGtIvfRrwePoelzgWHOlzwP5+8Dnn9n53oLPugy+5+CzfYs/MK6npgtaGOn6ks+XDj43+TTzVka6wUe5NNNWeqrhRwmwDVwHtm8ip/Erqan8Nx2fW+fnlSP9mtI6ri11f/Xa8rWQ0/nl0j+Df4R83/Aj1HUZ9hD+PH9+n3Jf/yPxwKhV+vnYA+NW8H5+ID00lWynkGbeRgXbkh+YtcmlmbTKpht/pN03b2OkW7TB9xi+1/inbS3b4P7B3yZ10ufw39DxGf7ta2/Hz0Ih/LkgEjjVpDBWEx7O/YjH89qIWH4blsRtI0Wy20gp4O+acNCmgjaM3YHVCnwEWrAwVgMRz6nEozjleBK3EO5X+rUQ5N8CD2UbditFH3tw4xKwEv5idPsHA4IgCIIgCIIgCIIgCPK/QU3X+yC9DEEQBEGQfw5yOh9+KasV3v+z6zrcFk/jSbb9K9sjCIIg/zk8lOWBwnMEQRAEQRAEQRAEQRAEQRAEQRAEQRDkXw2F5wiCIAiCIAiCIAiCIAiCIAiCIAiCIMi/HgrPEQRBEARBEARBEARBEARBEARBEARBkH89FJ4jCIIgCIIgCIIgCIIgCIIgCIIgCIIg/3ooPEcQBEEQBEEQBEEQBEEQBEEQBEEQBEH+9X43PJ/gtakvcY+3BmywHUEQBEEQBEEQBEEQBEEQBEEQBEEQBEH+r6OGcCeT0A3d0A3d0A3d0A3d0A3d0A3d0A3d0A3d0A3d0A3d0A3d0A3d0A3d0A3d0A3d/u23/weVqdwXr/GK8gAAAABJRU5ErkJggg=="
```

Comment:
Do we have a public brand image URL available for the Cortex PAN image?

Reply:
@andrew-paloalto , do you know for this one?
@BigEasyJ , if we add to our pack, how will automation reference it?

Reply:
nvm, I got what I needed from slack